charts/ia-home-office-integration-api/values.yaml

```diff
@@ -5,6 +5,9 @@ java:
   environment:
     IDAM_URL: "https://idam-api.{{ .Values.global.environment }}.platform.hmcts.net"
     S2S_URL: "http://rpe-service-auth-provider-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal"
+    HOME_OFFICE_ENDPOINT: "http://ia-home-office-mock-api-aat.service.core-compute-aat.internal"
+    HOME_OFFICE_CASE_SEARCH_URI: "/v1/applicationStatus/getBySearchParameters"
+    HOME_OFFICE_INSTRUCT_URI: "/v1/applicationInstruct/setInstruct"
```

Comment:
if endpoints are same for all environments, there is no need to put env variables here. It is ok to have default values in `application.yaml`

Reply:
I am keeping only HOME_OFFICE_ENDPOINT
New PR https://github.com/hmcts/ia-home-office-integration-api/pull/75

Reply:
application.yml points to localhost

Reply:
Marcin is talking about the paths on this line not the host.  If the paths are always the same for every environment (I think they are), you can put them directly in `application.yaml`

Reply:
This is changed in PR #77